medusa/providers/generic_provider.py

```diff
@@ -541,15 +541,15 @@ def _get_season_search_strings(self, episode):
             'Season': []
         }
 
-        for show_name in episode.show.get_all_possible_names(season=episode.season):
+        for show_name in episode.show.get_all_possible_names(season=episode.scene_season):
```

Comment:
Not that i'm trying to sneak this in here. But I noticed this as an inconsistency.